src/clj/game/cards-agendas.clj

```diff
@@ -254,7 +254,8 @@
                        :effect (effect (corp-install target nil {:install-state :rezzed}))} card targets))}
 
    "Mandatory Upgrades"
-   {:effect (effect (gain :click 1 :click-per-turn 1))}
+   {:effect (effect (gain :click 1 :click-per-turn 1))
+    :leave-play (effect (lose :click-per-turn 1))}
```

Comment:
Should the leave play not also lose a click? e.g. if the corp has to forfeit an agenda halfway through the turn, they should lose the 5th click. See also, [Rachel Beckman](https://github.com/mtgred/netrunner/blob/master/src/clj/game/cards-resources.clj#L441)